test/Microsoft.AspNet.Hosting.Tests/project.json

```diff
@@ -9,7 +9,6 @@
         "Microsoft.AspNet.Testing": "1.0.0-*",
         "Microsoft.Extensions.OptionsModel": "1.0.0-*",
         "Microsoft.Extensions.PlatformAbstractions": "1.0.0-*",
-        "Moq": "4.2.1312.1622",
```

Comment:
Done, the other one has it https://github.com/aspnet/Hosting/blob/pakrym/remove-moq/test/Microsoft.AspNet.TestHost.Tests/project.json#L17